.github/workflows/soak-testing.yml

```diff
@@ -32,21 +33,77 @@ env:
   # THROUGHPUT_PER_SECOND: TBD?
 
 jobs:
+  build_apps_to_ecr:
+    name: Publish Apps to ECR
```

Comment:
Soak tests should not be publishing anything, only main (snapshot) and release build.

I think we need to parameterize this to specify `tags = ...:alpha` for the main-build and can reference alpha (latest snapshot) here

https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/sample-apps/spark-awssdkv1/build.gradle.kts#L24

Reply:
Alternatively `jibDockerBuild` should work I think, but make sure to run it in the same `job` as the soak tests

Reply:
To speed up our solution here, we will "build" first and consider it a follow-up to "pull" the snapshot from the main build.

https://github.com/aws-observability/aws-otel-python/pull/26